<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1546</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Song</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>DDD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>880</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_do">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>890</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="codeImage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>160</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="background">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>1481</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backImage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="score">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>760</y>
     <width>881</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_train">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>820</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>backImage</zorder>
  <zorder>background</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>pushButton_do</zorder>
  <zorder>codeImage</zorder>
  <zorder>score</zorder>
  <zorder>pushButton_train</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>925</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>818</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_do</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1318</x>
     <y>893</y>
    </hint>
    <hint type="destinationlabel">
     <x>1208</x>
     <y>791</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_train</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>train()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1378</x>
     <y>842</y>
    </hint>
    <hint type="destinationlabel">
     <x>1391</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goBack()</slot>
  <slot>doSong()</slot>
  <slot>train()</slot>
 </slots>
</ui>
